File: php/auto_order/getAutoOrder.php
<?php

ini_set('display_errors', 1);

/*
 * retrieves an auto_order given the auto_order_oid;
 */

require_once '../vendor/autoload.php';
require_once '../samples.php';


$auto_order_api = Samples::getAutoOrderApi();

$_expand = "items,items.future_schedules,original_order,rebill_orders"; // see https://www.ultracart.com/api/#resource_auto_order.html for list
$auto_order_oid = 123456789; // If you don't know the oid, use getAutoOrdersByQuery for retrieving auto orders
$api_response = $auto_order_api->getAutoOrder($auto_order_oid, $_expand);
$auto_order = $api_response->getAutoOrder();
var_dump($auto_order);

File: python/auto_order/get_auto_order.py
from ultracart.apis import AutoOrderApi
from samples import api_client


# retrieves an auto_order given the auto_order_oid

def get_auto_order():
    auto_order_api = AutoOrderApi(api_client())

    # see https://www.ultracart.com/api/#resource_auto_order.html for list
    expand = "items,items.future_schedules,original_order,rebill_orders"

    # If you don't know the oid, use getAutoOrdersByQuery for retrieving auto orders
    auto_order_oid = 123456789
    api_response = auto_order_api.get_auto_order(auto_order_oid, expand=expand)
    auto_order = api_response.auto_order
    print(auto_order)


if __name__ == "__main__":
    get_auto_order()